internal/cluster/validation_id.go

```diff
@@ -32,6 +32,7 @@ const (
 	IsLsoRequirementsSatisfied          = ValidationID(models.ClusterValidationIDLsoRequirementsSatisfied)
 	IsCnvRequirementsSatisfied          = ValidationID(models.ClusterValidationIDCnvRequirementsSatisfied)
 	IsLvmRequirementsSatisfied          = ValidationID(models.ClusterValidationIDLvmRequirementsSatisfied)
+	IsMceRequirementsSatisfied          = ValidationID(models.ClusterValidationIDMceRequirementsSatisfied)
```

Comment:
where is the implementation of the validation?

Reply:
It is here: https://github.com/openshift/assisted-service/pull/4673/files#diff-6296026819131d76f0601cdfbb4635be91e8a228767cfc42d11341e952c3b8e4R101-R137. Currently it is empty.

Reply:
Well, that ^ is for the requirements. The validation itself is here: https://github.com/openshift/assisted-service/pull/4673/files#diff-6296026819131d76f0601cdfbb4635be91e8a228767cfc42d11341e952c3b8e4R66-R84. It is also empty.

Reply:
Maybe i'm missing something, i was  searching for something like `isNetworkTypeValid` the validation is used in https://github.com/openshift/assisted-service/blob/a328edfdc7fa049a03ba921de1c09705fa2f5533/internal/cluster/refresh_status_preprocessor.go#L131-L132
otherwise no one will run it no? unless it's wrapped somehow.

Reply:
I believe that the validations of the operator are executed separately, and then the results merged: https://github.com/openshift/assisted-service/blob/87e629425ea59d8ae3972b8037099feac006faf8/internal/cluster/refresh_status_preprocessor.go#L74-L78.

Reply:
ok thanks this is what i was missing.